sdk/servicebus/service-bus/samples/javascript/advanced/deferral.js

```diff
@@ -111,7 +111,7 @@ async function receiveMessage() {
     receiver.subscribe(
       { processMessage, processError },
       {
-        autoComplete: false
+        autoCompleteMessages: false
```

Comment:
streaming sample too

Reply:
I believe the streaming one does not have this setting yet... It is being added in your PR right?

Reply:
It is present in the comments at least!

Reply:
FINE!

https://github.com/Azure/azure-sdk-for-js/pull/12558/commits/d5369c7336273de4019e8317d39c7cccdf263964

:)